### ISSUE TITLE
Fixing Accessibility Issues

### ISSUE THREAD

**Issue by mgifford** [antennaio/jquery-bar-rating#113]
There are some outstanding accessibility issues which should be fixed.  Both empty links and undefined form elements. 

<img width="1034" alt="screen shot 2018-08-01 at 6 43 02 am" src="https://user-images.githubusercontent.com/116832/43517307-38cdd6fc-9556-11e8-9eb2-8fe8e2cabd94.png">

There needs to be additional semantics added to this to see that assistive technology users can properly know what they are clicking on.  

This is entirely based on evaluation of http://antenna.io/demo/jquery-bar-rating/examples/ using the Wave Toolbar http://wave.webaim.org/extension/

Adding more information about what the links are will give context to help users know what they are selecting.

**Comment by PSF1** [comment]
With the "showValues" option we can display the labels.

**Comment by PSF1** [comment]
The missing label tag maybe that was not inserted manually.

**Comment by PSF1** [comment]
I added a pull request with a partial solution.

https://github.com/antennaio/jquery-bar-rating/pull/123
